pkg/cmd/pipeline/describe.go

```diff
@@ -34,7 +34,7 @@ import (
 const describeTemplate = `{{decorate "bold" "Name"}}:	{{ .PipelineName }}
 {{decorate "bold" "Namespace"}}:	{{ .Pipeline.Namespace }}
 
-{{decorate "underline bold" "Resources\n"}}
+{{decorate "resources" ""}}{{decorate "underline bold" "Resources\n"}}
```

Comment:
tried to do stuff like this : 

```
{{ decorate "inputresources" "" | decorate "underline bold" "Input Resources\n"}}
``

but I am getting the error message : 

```
Error: template: Describe Task:4:33: executing "Describe Task" at <decorate>: wrong number of args for decorate: want 2 got 3
```
🤔